scripts/tools/EXT_Exp_MetricProgrammable.rst

```diff
@@ -0,0 +1,123 @@
+<%
+import re
+from templates import helper as th
+%><%
+    OneApi=tags['$OneApi']
+    x=tags['$x']
+    X=x.upper()
+    t=tags['$t']
+    T=t.upper()
+%>
+:orphan:
+
+.. _ZET_experimental_programmable_metric:
+
+==========================================
+Metric Programmable Experimental Extension
+==========================================
+
+API
+----
+* Enumerations
+
+    * ${t}_metric_programmable_param_type_exp_t
+    * ${t}_value_info_type_exp_t
+    * ${t}_metric_programmable_exp_version_t
+
+* Structures
+
+    * ${t}_metric_programmable_exp_properties_t
+    * ${t}_metric_programmable_param_info_exp_t
+    * ${t}_metric_programmable_param_value_exp_t
+    * ${t}_metric_programmable_param_value_info_exp_t
+    * ${t}_value_info_exp_t
+    * ${t}_value_uint64_range_exp_t
+
```

Comment:
zet_metric_programmable_param_description_t ? needed by zetMetricProgrammableGetParamInfo()

Reply:
It has been renamed with "info", since description has a different meaning in other places of L0 spec
zet_metric_programmable_param_info_exp_t